packages/web3-core/package.json

```diff
@@ -14,8 +14,8 @@
 		"prebuild": "rimraf dist",
 		"build": "tsc --build",
 		"build:check": "node -e \"require('./dist')\"",
-		"lint": "eslint --ext .js,.ts .",
-		"lint:fix": "eslint --fix --ext .js,.ts .",
+		"lint": "eslint --ext .js,.ts . --rulesdir ../../tools/eslint-config-web3-base/rules",
+		"lint:fix": "eslint --fix --ext .js,.ts . --rulesdir ../../tools/eslint-config-web3-base/rules",
```

Comment:
It's a bad practice to reference a file outside package scope. If there is no other workaround I would suggest to create symlinks for this directories and reference local directory in the package.